packages/@react-spectrum/list/src/index.css

```diff
@@ -0,0 +1,41 @@
+.react-spectrum-List {
+  border-color: var(--spectrum-table-border-color, var(--spectrum-alias-border-color-mid));
+  border-style: solid;
+  position: relative;
+  border-width: var(--spectrum-table-border-size, var(--spectrum-alias-border-size-thin));
+  border-radius: var(--spectrum-table-border-radius, var(--spectrum-alias-border-radius-regular));
+  overflow: auto;
+  vertical-align: var(--spectrum-table-cell-vertical-alignment);
+  border-collapse: separate;
+  border-spacing: 0;
+  transform: translate3d(0, 0, 0);
+  padding: 0;
+  background-color: var(--spectrum-table-background-color, var(--spectrum-global-color-gray-50));
+  outline: 0;
+}
+
+.react-spectrum-ListItem {
```

Comment:
Are we going to go with the slots-based approach for items similar to ListBox? In that case we'll want to define a CSS grid here and some slots for content to go in. Also fine to do that later.